src/Models/Message.php

```diff
@@ -114,11 +168,15 @@ public function getAddressee(): CharacterInterface
     
     /**
      * Sets the author of this motd
```

Comment:
I really should look at the comments as well if i'm copy pasting. No, the description is wrong, this is the Message model.